docs/src/adjoints.md

```diff
@@ -1,11 +1,13 @@
 # Custom Adjoints
 
-!!! note "Prefer to use ChainRules to define custom adjoints"
+!!! note "Prefer to use ChainRulesCore to define custom adjoints"
     Zygote supports the use of [ChainRulesCore](http://www.juliadiff.org/ChainRulesCore.jl/stable/) to define custom sensitivities.
     It is prefered to define the custom sensitivities using `ChainRulesCore.rrule` as they will work for many AD systems, not just Zygote.
-    These sensitivities can be added in your own package, or for Base functions they can be added to ChainRules.jl.
-
-    This documentation exists to descibe how Zygote works, and how adjoints can be directly defined for Zygote.
+    These sensitivities can be added in your own package, or for Base/StdLib functions they can be added to [ChainRules.jl](https://github.com/JuliaDiff/ChainRules.jl/).
+    To use define custom sensitivities using ChainRulesCore, define a `ChainRulesCore.rrule(f, args...; kwargs...)` [ChainRules project's documentation for more information](https://www.juliadiff.org/ChainRulesCore.jl/stable/).
+    **If using ChainRulesCore you do not need to read this page**, and can consider it as documenting a legacy feature.
```

Comment:
I think it's a bit confusing at the offset whether I am using CRC, (I just did `using Zygote`, where does CRC come from?)

Reply:
```suggestion
    **If you are defining your custom adjoints using ChainRulesCore then you do not need to read this page**, and can consider it as documenting a legacy feature.
```